zstacklib/zstacklib/utils/singleflight.py

```
@@ -1,5 +1,7 @@
 import threading
+import time
 
+logger = log.get_logger(__name__)
```

Ruff (0.8.2)

Undefined name `log`

(F821)